tests/postgresl_flexible/test_postgresql_flexible_actions.py

```diff
@@ -5,20 +5,21 @@
 
 import chaosazure
 from chaosazure.postgresql_flexible.actions import restart_servers, stop_servers, \
-    delete_servers, start_servers
+    delete_servers, start_servers, delete_tables
 
 from data import postgresl_flexible_provider, config_provider, secrets_provider
 
 CONFIG = {
-    "azure": {
-        "subscription_id": "***REMOVED***"
-    }
+    "azure_subscription_id": "***REMOVED***"
 }
 
 SECRETS = {
-    "client_id": "***REMOVED***",
-    "client_secret": "***REMOVED***",
-    "tenant_id": "***REMOVED***"
+    "azure": {
```

Comment:
I followed the directions in the readme:

"{
  "secrets": {
    "azure": {
      "client_id": "your-super-secret-client-id",
      "client_secret": "your-even-more-super-secret-client-secret",
      "tenant_id": "your-tenant-id"
    }
  }
}"

I thought it would be useful to move the tests in this direction. Since then, I've been retrieving my identifiers from the "azure" sub-dictionary in my "delete_tables" action

what do you think about this ?

Reply:
That makes sens :)